THIRD-PARTY-NOTICE.txt

```diff
@@ -0,0 +1,1998 @@
+WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+DAMAGE.
+
+License notice for Cglib
+------------------------------------------------------------------------------
+
+ Copyright 2003,2004 The Apache SoftwareFoundation
+ Licensed under the Apache License, Version 2.0 (the"License");
+ you may not use this file except in compliance with theLicense.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,software
+ distributed under the License is distributed on an "AS IS"BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express orimplied.
+ See the License for the specific language governing permissionsand
+ limitations under the License.
+
+License notice for Jackson Dataformat
+------------------------------------------------------------------------------
+
+This copy of Jackson JSON processor databind module is licensed under the
+Apache (Software) License, version 2.0 ("the License").
+See the License for details about distribution rights, and the
+specific rights regarding derivate works.
+
+You may obtain a copy of the License at:
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+License notice for Jackson Datatype
+------------------------------------------------------------------------------
+
+This copy of Jackson JSON processor streaming parser/generator is licensed under the
+Apache (Software) License, version 2.0 ("the License").
+See the License for details about distribution rights, and the
+specific rights regarding derivate works.
+
+You may obtain a copy of the License at:
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+License notice for Wiremock
+------------------------------------------------------------------------------
+
+Copyright (C) 2011 Thomas Akehurst
+
+This copy of Jackson JSON processor databind module is licensed under the
+Apache (Software) License, version 2.0 ("the License").
+See the License for details about distribution rights, and the
+specific rights regarding derivate works.
+
+You may obtain a copy of the License at:
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+License notice for Google Findbugs
+------------------------------------------------------------------------------
+
+   Copyright (C) 2003-2008 University of Maryland
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+License notice for Nimbusds
```

Comment:
Nimbusds has a lot of projects under it, like Jackson - please be more specific.

Reply:
Done.